### PR TITLE
Fluxcd support self signed repo

### PR DESCRIPTION
**Pull request by wangyikewxgm**
### Checklist

I have:

- [ ] Title of the PR starts with type (e.g. `[Addon]` , `[example]` or `[Doc]`).
- [ ] Updated/Added any relevant [documentation](https://kubevela.io/docs/reference/addons/overview) and [examples](../examples).
- [ ] New addon should be put in [experimental](../experimental/addons).
- [ ] Update addon should modify the `version` in `metadata.yaml` to generate a new version.
- [ ] Any changes about [verified](../addons) addons should be tested with CI [script](../test/e2e-test/hack/addon-vela-test.sh).

### CODE REVIEW

**Review by wonderflow — APPROVED** [review]

File: addons/fluxcd/definitions/config-helm-repository.yaml [hunk 1]
```diff
@@ -37,16 +37,20 @@ spec:
...
         	// +usage=The public url of the helm chart repository.
         	url: string
-        	// +usage=The username of basic auth repo
+        	// +usage=The username of basic auth repo.
         	username?: string
-        	// +usage=The password of basic auth repo
+        	// +usage=The password of basic auth repo.
         	password?: string
+        	// +usage=The ca certificate of helm repository.
+        	caFile?: string
```
Comment (barnettZQG) [inline addons/fluxcd/definitions/config-helm-repository.yaml:53]: Is this a CA file content?

Comment (wangyikewxgm) [reply addons/fluxcd/definitions/config-helm-repository.yaml:53]: Yes，refer to： https://fluxcd.io/docs/components/source/helmrepositories/#tls-authentication

**Review by FogDong — APPROVED** [review]

File: addons/fluxcd/definitions/config-helm-repository.yaml [hunk 2]
```diff
@@ -37,16 +37,22 @@ spec:
         		if parameter.password != _|_ {
         			password: parameter.password
         		}
-
+        	}
+        	data: {
+        		if parameter.caFile != _|_ {
+        			caFile: parameter.caFile
```
Comment (FogDong) [inline addons/fluxcd/definitions/config-helm-repository.yaml:43]: Can we get this from secret?

Comment (wangyikewxgm) [reply addons/fluxcd/definitions/config-helm-repository.yaml:43]: The target of this componentDefinition is to generate a secret. This is a type of integration.

**Pull request merged**